### DISCUSSION TITLE
Grid.js with JQuery

### DISCUSSION THREAD

**Discussion by johnstaveley** [grid-js/gridjs#1156] [Q&A]
I am importing grid.js with jquery from the CDN:

`<script src="https://unpkg.com/gridjs-jquery/dist/gridjs.production.min.js"></script>`

How do I do html and h elements such as described in (https://gridjs.io/docs/examples/row-buttons/) and
 (https://gridjs.io/docs/examples/html-cells)
without creating a module? It says h and html are undefined otherwise.

**Comment by Nicezki** [comment]
Since I also have a problem and my solution is using gridjs.h instead of h
For me I use umd version

Example:
From
```
const editButton = h('button', {
          className: 'py-2 px-4 mx-2 border rounded-md text-white bg-blue-600',
          onClick: () => alert(`Editing "${row.cells[1].data}"`)
}, 'Edit');
```

to
```
const editButton = gridjs.h('button', {
          className: 'py-2 px-4 mx-2 border rounded-md text-white bg-blue-600',
          onClick: () => alert(`Editing "${row.cells[1].data}"`)
}, 'Edit');
```
